### DISCUSSION TITLE
Jump to start of astronomical night for current location

### DISCUSSION THREAD

**Discussion by kopi23** [Stellarium/stellarium#2607] [Q&A] [answered]
Hi,

is it possible to jump in time to the start of the astronomical night for the current location?

Thank you, kopi

**Comment by alex-w** [comment]
This is possible through scripts or hotkeys.

**Reply by kopi23** [reply]
thanks alex-w. i found a few scripts with the 'Dawn/dusk/sunrise/sunset jumps' (http://www.alienbasecamp.com/Stellarium/sun.htm) from Qam1 coming closest i guess. but it does so much more than just fast forward to astronomical night.
do you might know of a script doing just that?

**Comment by gzotti** [comment, marked as answer]
See User Guide, 4.8.1

**Reply by kopi23** [reply]
thanks gzotti. i saw them and was wondering if such functionality would also exist for astronomical twilight.
or maybe the depth of the sun below the horizon is configurable for this build in functionality? so i could set it from -6° to -18°?

**Reply by gzotti** [reply]
See "Solar altitude for Twilight Finder" on page 29.

**Reply by kopi23** [reply]
thank you gzotti. works brilliantly.
